### ISSUE TITLE
List Inconsolata-Regular as a dependency

### ISSUE THREAD

**Issue by felipesanches** [graphicore/librebarcode#8]
The build script seems to use Inconsolata-Regular.ttf for the --fontbelow feature in the "Text" variants for rendering text below the barcodes.

There's the issue that the build script assumes Inconsolata is available at this relative directory:
```
TEXT_BELOW_FONT=`pwd`/../fonts/ofl/inconsolata/Inconsolata-Regular.ttf
```

And I also wonder whether wouldn't it be good to mention somewhere that the resulting font includes glyphs from Inconsolata-Regular and thus is a derivative of it.

**Comment by davelab6** [comment]
I agree that

    TEXT_BELOW_FONT=`pwd`/../fonts/ofl/inconsolata/Inconsolata-Regular.ttf

is an easy stumbling point; could be good to just copy a version of that font into the repo and refer to its local path

**Comment by khaledhosny** [comment]
This was fixed in 1477c7c08296123aa6ac803647fd54e5e97aaad0 by copying `Inconsolata-Regular.ttf` to the repository.
